controllers/socketController.js

```diff
@@ -476,8 +483,8 @@ module.exports = function (io) {
                 if (userWasAwake) {
                     io.to(location).emit('sleep', { userToSleep });
                 } else {
-                    io.to(socket.id).emit('error', { message: "You are already sleeping" });
```

Comment:
Hey the lines of code that were overwritten here are incorrect. I believe they were accidentally deleted during a merge conflict. This is the code that needs to stay:
```
io.to(socket.id).emit('error', { message: "You are already sleeping" });
io.to(socket.id).emit('sleep', { userToSleep, quiet: true });
```